src/Pages/Webhooks.php

```diff
@@ -237,7 +240,7 @@ protected function isTablePaginationEnabled(): bool
 
     protected function getTableEmptyStateIcon(): ?string
     {
-        return 'gmdi-webhook';
+        return config('filament-webhook-server.navigation.icon');
```

Comment:
Can you add some validation if the `navigation.icon` is existing in the config, and if not return the icon name instead.